machine/machine.go

```diff
@@ -122,6 +124,44 @@ func GetClockSpeed(procInfo []byte) (uint64, error) {
 	return uint64(speed * 1000), nil
 }
 
+func GetVMStats(vmStatMetrics *string) (map[string]int, error) {
+	vmstatRegexp, err := regexp.Compile(*vmStatMetrics)
+	if err != nil {
+		return nil, fmt.Errorf("failed to compile vmstat regexp: %v", err)
+	}
+
+	stats := make(map[string]int)
+	file, err := os.Open(vmstatFile)
+	if err != nil {
+		return nil, fmt.Errorf("cannot open %s", vmstatFile)
+	}
+	scanner := bufio.NewScanner(file)
+
+	for scanner.Scan() {
+		splittedLine := strings.SplitN(scanner.Text(), " ", 3)
+		if len(splittedLine) < 2 {
+			klog.Info("cannot parse vmstat's metric. Skipping")
```

Comment:
It seems that it's worth to add information about problematic line, it will be easier to debug potential issue.